ui/src/app/pages/admin/user-audit.tsx

```diff
@@ -0,0 +1,46 @@
+import {profileApi} from 'app/services/swagger-fetch-clients';
+import * as fp from 'lodash/fp';
+import * as React from 'react';
+import {useParams} from 'react-router-dom';
+import {AuditAction, AuditEventBundle} from '../../../generated';
+import {AuditPageComponent} from '../../components/admin/audit-page-component';
+
+const getAuditLog = (subject: string) => {
+  const bqRowLimit = 1000; // Workspaces take many rows because of the Research Purpose fields
+  return profileApi().getAuditLogEntries(subject, bqRowLimit);
+};
+
+const queryAuditLog = (subject: string) => {
+  return getAuditLog(subject).then((queryResult) => {
+    console.log(queryResult.query);
+    return {
+      actions: queryResult.actions,
+      sourceId: queryResult.userDatabaseId,
+      query: queryResult.query,
+      logEntries: queryResult.logEntries
+    };
+  }).then(genericQueryResult => {
+    // TODO(jaycarlton): This is a workaround for LOGIN event issues on the backend. Can be removed when that patch is in.
```

Comment:
what ticket captures this work?

Reply:
https://precisionmedicineinitiative.atlassian.net/browse/RW-5249

Reply:
sorry, I wasn't clear - can you put that information in the comment?